devlib/utils/android.py

```diff
@@ -294,6 +294,10 @@ def __init__(
             total_transfer_timeout=total_transfer_timeout,
             transfer_poll_period=transfer_poll_period,
         )
+
+        self.logger.debug('AdbConnection(): server=%s port=%s device=%s as_root=%s',
```

Comment:
s/AdbConnection(): //

The name of the class should be used when the logger is made.